asyn/drvAsynSerial/drvAsynSerialPortWin32.c

```diff
@@ -61,6 +61,7 @@ typedef struct {
     double             writeTimeout;
     epicsTimerId       timer;
     volatile int       timeoutFlag;
+    unsigned           sleep_len;     /* length of sleep after sending bytes (ms). If both are defined sleep happens before break. */
     unsigned           break_len;     /* length of serial break to send after a write (ms) */
```

Comment:
Would it be better to name these the same as the asyn parameters i.e. break_duration and break_delay?